package com.lumbre.auth.exception

class UserNotFoundException(message: String) : RuntimeException(message) {
}